src/Platform.Handlers/src/Xamarin.Platform.Handlers/ActionMapper.cs

```diff
@@ -1,22 +1,24 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using Xamarin.Platform.Handlers;
 
 namespace Xamarin.Platform
 {
-	public class ActionMapper<TVirtualView>
+	public class ActionMapper<TVirtualView, TViewHandler>
```

Comment:
don't you want a constraint on TViewHandler being IViewHandler ?

Reply:
https://github.com/xamarin/Xamarin.Forms/pull/12198/files/e2b03332fad9781efafe5b5b853d275374a262f7#diff-d7ba16578d6228dae25138978651ce28R10